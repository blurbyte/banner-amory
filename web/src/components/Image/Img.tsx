import styled from 'styled-components';

const Img = styled.img`
  max-width: 100%;
  user-select: none;
  position: absolute;
  z-index: 9;
  top: 0;
  left: 0;
`;

export default Img;
