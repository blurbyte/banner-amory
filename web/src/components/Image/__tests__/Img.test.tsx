import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Img from '../Img';

test('renders correctly', () => {
  process.env.STORAGE_URL = '/test';
  const component = renderer.create(<Img />);
  expect(component.toJSON()).toMatchSnapshot();
});
