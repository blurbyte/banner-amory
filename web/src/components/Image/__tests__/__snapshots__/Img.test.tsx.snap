// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  max-width: 100%;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: absolute;
  z-index: 9;
  top: 0;
  left: 0;
}

<img
  className="c0"
/>
`;
